### PR TITLE
Update humans.txt with Gareth

### PR DESCRIPTION
**Pull request by gtrufitt**
# ARE YOU A HUMAN?

yes1
# DO YOU WORK ON FRONTEND?

yes2

Well, how about you add yoself to our [humans.txt](http://humanstxt.org/), the canonical reference for all the people that contribute, behind-the-scenes, to our glorious glorious website.

Push to this branch.

@guardian/guardian-frontend @guardian/dotcom-platform @guardian/commercial-dev @guardian/participation  @guardian/membership-and-subscriptions

### CODE REVIEW

File: facia/public/humans.txt [hunk 1]
```diff
@@ -195,3 +195,8 @@ Location: London, UK
 
 Developer: Jamie Byers
 Location: London, UK
+
```
Comment (NataliaLKB) [inline facia/public/humans.txt]: You put yourself in Alumni!!

Comment (gtrufitt) [inline facia/public/humans.txt]: I wasn't really sure whether *team* was the original team and *Alumni and Colleagues* was everyone else. Maybe I can be counted as *team* 🎈

**Comment by NataliaLKB** [comment]
Once you move yourself out of Alumni 👍 . If you have time, could you also move the people that aren't here anymore down to Alumni?

The ones I spotted were: 
* Olly https://github.com/guardian/frontend/blob/27f11fbb59e648150b8de8e270c23da465896c98/facia/public/humans.txt#L64
* Shannon https://github.com/guardian/frontend/blob/27f11fbb59e648150b8de8e270c23da465896c98/facia/public/humans.txt#L60
* Wolfgang https://github.com/guardian/frontend/blob/27f11fbb59e648150b8de8e270c23da465896c98/facia/public/humans.txt#L57

Other Changes could be:
* Stephan had a role change https://github.com/guardian/frontend/blob/27f11fbb59e648150b8de8e270c23da465896c98/facia/public/humans.txt#L26

File: facia/public/humans.txt (hunk 1 above)
Comment (desbo) [inline facia/public/humans.txt]: Hey, I'm in alumni as well!
This needs rearranging, alumni and colleagues is surely meant to be for people who've left.

Comment (gtrufitt) [inline facia/public/humans.txt]: Moved myself and Sam

**Comment by johnduffell** [comment]
I'm sure stephan won't mind, but since this is personal data, people are supposed to add/edit their own entries
other than that, 👍

**Comment by gtrufitt** [comment]
> people are supposed to add/edit their own entries

Good point, have reverted the @stephanfowler change, he can update his own tomorrow

File: facia/public/humans.txt [hunk 2]
```diff
@@ -1,4 +1,9 @@
-/* TEAM */
+/* CURRENT TEAM */
+
+Chief Coder: Frank Hulley-Jones
```
Comment (gtrufitt) [inline facia/public/humans.txt]: Frank you are cruising to be changed to `intern colourer-inner` here...

**Comment by zeftilldeath** [comment]
🌊 🌴 🌞  This PR rules! 🌞 🌴 🌊

File: facia/public/humans.txt (hunk 2 above)
Comment (zeftilldeath) [inline facia/public/humans.txt]: He's cruisin' for a bruisin'

**Comment by janua** [comment]
snaps digit nails? :scream:

**Comment by piuccio** [comment]
I'm on holiday for a week,  can't push :cry:

**Comment by NataliaLKB** [comment]
@piuccio What do you want it to say? Happy to do it for you!

File: facia/public/humans.txt [hunk 3]
```diff
@@ -50,23 +59,73 @@ From: Halifax, Canada
...
 Twitter: @asanders
 
-Director of Digital Strategy: Wolfgang Blau
-Twitter: @wblau
+Developer: John Duffell
 
-Guardian CTO: Shannon Maher
+Developer: Zofia Korcz
+Twitter: @intuitio
+From: Poznan, Poland
```
Comment (paperboyo) [inline facia/public/humans.txt]: Ahem! `Poznań`

rogal means grin

File: facia/public/humans.txt [hunk 4]
```diff
@@ -193,5 +251,15 @@ Twitter: @jbreckmckye
 Website: www.breck-mckye.com
 Location: London, UK
 
-Developer: Jamie Byers
-Location: London, UK
+Developer: Oliver Ash
+Twitter: @OliverJAsh
+From: London, UK
+
+Engineer / Dev Manager: James Pamplin
```
Comment (jamespamplin) [inline facia/public/humans.txt]: Added myself in an adequate section.

**Pull request merged**